nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -134,6 +147,18 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
             elif self.cfg.strategy in ['beam', 'tsd', 'alsd', 'maes']:
                 self.preserve_alignments = self.cfg.beam.get('preserve_alignments', False)
 
+        # Update compute timestamps
+        if self.compute_timestamps is None:
+            if self.cfg.strategy in ['greedy', 'greedy_batch']:
+                self.compute_timestamps = self.cfg.greedy.get('compute_timestamps', False)
+
+            elif self.cfg.strategy in ['beam', 'tsd', 'alsd', 'maes']:
+                self.compute_timestamps = self.cfg.beam.get('compute_timestamps', False)
+
+        # Test if alignments are being preserved for RNNT
+        if self.compute_timestamps is True and self.preserve_alignments is False:
```

Comment:
Can we just enable the preserve_alignments automatically when compute_timestamps is requested?

Reply:
Hmm possible. I'll look into it when I get back.

Reply:
Transducer is RNNT in Nemo. And CTC support is also there.